internal/freelist/freelist.go

```diff
@@ -0,0 +1,108 @@
+package freelist
+
+import (
+	"sort"
+
+	"go.etcd.io/bbolt/internal/common"
+)
+
+// FreelistType is the type of the freelist backend
+type FreelistType string
```

Comment:
we might need to keep those in the bbolt package, or copy&paste them potentially

Reply:
moved it back to the bbolt package

Reply:
Right, we can't break the user experience in 1.4.